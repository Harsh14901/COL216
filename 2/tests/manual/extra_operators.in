19*4-69*--2-68-9***
194581++9+8*+5++-**
71*6*23++527*-*46*-+
516--221+60-39--+--*
37+9+-73+*-0*251*0-*-
555--68*8**87*45*+--++
26082530*35*--+*-*-***
19*43-+*-55*+7266*-+*
86*778+6379+*-2*+***-
89-04-*972-+*-06-+*5-+*
