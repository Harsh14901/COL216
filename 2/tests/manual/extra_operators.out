[*] Executing test case - 19*4-69*--2-68-9***
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 194581++9+8*+5++-**
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 71*6*23++527*-*46*-+
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 516--221+60-39--+--*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 37+9+-73+*-0*251*0-*-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 555--68*8**87*45*+--++
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 26082530*35*--+*-*-***
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 19*43-+*-55*+7266*-+*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 86*778+6379+*-2*+***-
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
[*] Executing test case - 89-04-*972-+*-06-+*5-+*
[*] Correct --- Expected Error, Got : Error
    ====================================================================================================
